src/c2goto/c2goto.cpp

```diff
@@ -61,7 +61,7 @@ class c2goto_parseopt : public parseoptions_baset, public language_uit
     if(config.set(cmdline))
       return 1;
     config.options.cmdline(cmdline);
-    messaget::state.verbosity = VerbosityLevel::Result;
+    messaget::state.verbosity = VerbosityLevel::Debug;
```

Comment:
Could you remove this change? On a side note: I've also been using `Debug` here quite a lot, we should at some point in the future add a `--verbosity` flag for `c2goto`...